extensions/ql-vscode/src/cli.ts

```diff
@@ -1276,7 +1279,7 @@ export class CliVersionConstraint {
    /**
     * CLI version that supports rotating structured logs to produce one per query.
     */
-    public static CLI_VERSION_WITH_PER_QUERY_EVAL_LOG = new SemVer('2.8.4');
+    public static CLI_VERSION_WITH_PER_QUERY_EVAL_LOG = new SemVer('2.9.0');
```

Comment:
Why the change of version here? I believe everything we need for this is in 2.8.4.

Reply:
Ah, I got a bit mixed up with all the different PRs — the incoming versions to log to the query server output (that I had been planning to merge into your PR) will be in 2.9.0, but it does seem unnecessary to wait until 2.9.0 for these changes. 

So if a user is using 2.8.4, they would see the commands (and log files) but not any output in the query server console; and if they are using 2.9.0+ they would see the query server console output as well. I think that is reasonable (without adding another gate for 2.9.0).